Dockerfile

```diff
@@ -19,6 +19,7 @@ ENV PROXY_HOST=localhost \
 WORKDIR /app
 
 ADD requirements.txt /app/
+ADD docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
Should move this towards the end, after all the installs, to avoid rebuilding if this needs to change.

Reply:
Ok, reordered!